caluma/caluma_logging/tests/test_logging_middleware.py

```diff
@@ -0,0 +1,115 @@
+import pytest
+from django.urls import reverse
+
+from .. import models
+
+
+@pytest.mark.parametrize(
+    "query,variables,operation,operation_name,selection,status_code",
+    [
+        (
+            """
+            query foo {
```

Comment:
I'd probably factor out the query literals here so the parametrisation list becomes a bit more readable. Put it in a couple module-level variables for example.

Reply:
I agree it's not pretty but supporting different levels of nesting etc with parametrization would be too complicated to be useful.

Reply:
No i mean just put the content of the query string into a variable, so the parametrization list looks a bit cleaner. structure-wise it'll still be the same